builtin/providers/aws/resource_aws_s3_bucket_object.go

```diff
@@ -74,6 +75,11 @@ func resourceAwsS3BucketObject() *schema.Resource {
 				ConflictsWith: []string{"source"},
 			},
 
+			"kms_key_id": &schema.Schema{
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
 			"etag": &schema.Schema{
 				Type: schema.TypeString,
 				// This will conflict with SSE-C and SSE-KMS encryption and multi-part upload
```

Comment:
I got that covered in the next few lines. Is this ready to roll?

Sent from my iPhone

> On Mar 10, 2016, at 1:27 PM, Paul Hinze <notifications@github.com> wrote:
> 
> In builtin/providers/aws/resource_aws_s3_bucket_object.go:
> 
> > @@ -74,6 +75,11 @@ func resourceAwsS3BucketObject() *schema.Resource {
> >  				ConflictsWith: []string{"source"},
> >  			},
> >  
> > +			"kms_key_id": &schema.Schema{
> > +				Type:     schema.TypeString,
> > +				Optional: true,
> > +			},
> > +
> >  			"etag": &schema.Schema{
> >  				Type: schema.TypeString,
> >  				// This will conflict with SSE-C and SSE-KMS encryption and multi-part upload
> Oooooh you're right - I had forgotten!
> 
> Okay so we have to fall back to docs or checks within Create/Update like you said.
> 
> —
> Reply to this email directly or view it on GitHub.
>